circuits/core/manager.py

```diff
@@ -894,6 +894,7 @@ def processTask(self, event, task, parent=None):  # noqa
                 self._eventDone(event)
         except KeyboardInterrupt:
             self.stop()
+            raise
```

Comment:
That is mainly for now to let the error bubble up so I can exit pytest quickly (see note2 above)